tests/robot/OBSERVATION/04_create_heart_rate.robot

```diff
@@ -21,8 +21,11 @@ Resource                ${EXECDIR}/robot/_resources/suite_settings.robot
 
 Test Setup              generic.prepare new request session    Prefer=return=representation
 ...															   Authorization=Basic bXl1c2VyOm15UGFzc3dvcmQ0MzI=
+Documentation           NOTE: The validation of error messages in the tables below depends on system language settings:
```

Comment:
@af-dpe  the only thing I changed was to move this line of documentation into one place.